### PR TITLE
W02 sf - Bio Revisions

### PR DESCRIPTION
**Pull request by darkmaterials**

### CODE REVIEW

File: Individual Assignments/Fan Serenity/week-02/README.md [hunk 1]
```diff
@@ -2,3 +2,22 @@ Serenity Fan (they/them) is a 1st-year Georgetown student in the MIDP (Master of
 
 As an interdisciplinary data analyst, project manager, fundraiser, Serenity has over 4 years of professional experience - including 8 months Global South field experience - spanning rural water supply in Myanmar, immunization behaviour change in India, and refugee mental health in Canada. They have conducted scientific research doing data analysis, cleaning, and visualization in 4 research labs, in which capacity they developed proficiency in MATLAB, ArcGIS, and LaTeX/Overleaf. 
 
+My WASH Research Interests 
+-------------
+One area Serenity is particularly interested in within WASH is the intersection between WASH and public health. Many WASH programs are implemented under the tacit assumption that they will improve health outcomes -for instance, reducing diarrhoeal incidence for <5 children- but they do not necessarily have monitoring and evaluation systems and programs in place to ensure that these health outcomes occur. One organization they are interested in that is doing evidence-based work in this space is [Dispensers for Safe Water](https://www.evidenceaction.org/dispensersforsafewater/), which is housed within the INGO Evidence Action.
```
Comment (alexacoon) [inline Individual Assignments/Fan Serenity/week-02/README.md:7]: Super cool work. Is there any other organization that does similar work that you are interested in?

File: Individual Assignments/Fan Serenity/week-02/README.md [hunk 2]
```diff
@@ -2,3 +2,22 @@ Serenity Fan (they/them) is a 1st-year Georgetown student in the MIDP (Master of
...
+The table below provides a definitional overview of a hierarchy of designations of water quality. 
+
+![Alt text](\img\WASH-table.png)
+
+The UNHCR water supply standards (2007) provide another example of WASH standards in post-emergency contexts. In these standards, villages are defined as high-priority and water-stressed in which:  
+- Existing wells are non-functioning or producing inadequate water yield (less than 0.2 Litres/second), 
+- No alternative safe water source exists within at least 500 metres for the most distance household,
+- More than 250 people are using the same well, or 
+- Existing wells are close to contaminant sources such as stockyards, cemeteries, or latrines. 
+
```
Comment (alexacoon) [inline Individual Assignments/Fan Serenity/week-02/README.md:21]: Additions are great, maybe a concluding sentence?

**Review by alexacoon — APPROVED** [review]
Looks good!

**Review by darkmaterials — COMMENTED** [review]
Thank you for your comments. :)

**Pull request merged**